src/annotation.js

```diff
@@ -25,15 +25,17 @@ export default {
 	},
 
 	beforeUpdate(chart, args, options) {
+		const annotationOptions = options || args;
 		if (!args.mode) {
-			bindAfterDataLimits(chart, options);
+			bindAfterDataLimits(chart, annotationOptions);
 		}
 	},
 
 	afterUpdate(chart, args, options) {
+		const annotationOptions = options || args;
```

Comment:
why the `options || args`?

Reply:
because if I use options (therefore the third argument) Ive got the following exception (using the samples):`

```javascript
Uncaught TypeError: Cannot read property 'annotations' of undefined
    at getAnnotationOptions (chartjs-plugin-annotation.js:636)
    at adjustScaleRange (chartjs-plugin-annotation.js:640)
    at LinearScale.scale.afterDataLimits (chartjs-plugin-annotation.js:626)
    at LinearScale.update (chart.js:5459)
    at fitBoxes (chart.js:2839)
    at Object.update (chart.js:2951)
    at Chart._updateLayout (chart.js:7123)
    at Chart.update (chart.js:7104)
    at new Chart (chart.js:6881)
    at window.onload (bar.html:67)
```
I think this is related to this PR https://github.com/chartjs/Chart.js/pull/7949 where `args` argument has been added but I don't think it's included into chart.js version in node_modules.

```html
<script src="../node_modules/chart.js/dist/chart.js"></script>
```

In fact, if you are using CHART.JS dist/master it works.

```html
<script src="https://www.chartjs.org/dist/master/chart.min.js"></script>
```

Reply:
Right, I've been using it with master (`npm link` in chart.js and `npm link chart.js` in plugin to get master in node_moduled)

Reply:
To be removed when new CHART.JS will be published